### DISCUSSION TITLE
[IDEAS]Bumpkin inventory

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1862] [Ideas]
Just to illustrate an idea.

![image](https://user-images.githubusercontent.com/74167741/207279336-b95d6251-cf27-482b-a04f-a751e58dbd05.png)
![image](https://user-images.githubusercontent.com/74167741/207279385-a173cfa4-652b-4bbf-a0f7-f4bebc825315.png)
![image](https://user-images.githubusercontent.com/74167741/207279413-39aa99a0-110b-4a0f-90a7-2c672fb0a638.png)
![image](https://user-images.githubusercontent.com/74167741/207279434-5647cfdb-bf23-41bd-95bc-eab8e7798e2a.png)

I would also add a section for NFTS and active items on farm.

I also believe that it would be great to be able to exchange equipped items within the game without synchronizing on the blockchain, it would be fairer with those who have the 3 necklaces for example.

Thank you.

**Comment by Galphath** [comment]
I like it, Could also be nice to have some sort of first (default tab) tab where we could display our favorite 5 achievements, nfts and or skills

**Comment by Myth727** [comment]
Love this, they also need to add a count of inventory when cooking. For example I know it cost 5 eggs for boiled eggs, but I also wanna see how many I have without looking at inventory. 

![Screenshot_20221214_130531_MetaMask](https://user-images.githubusercontent.com/96928557/207673306-e39261aa-4daf-4e9f-a79d-3bb144c5d743.jpg)


This will help when a user is about to run out and not paying attention.  They might not cook food any item after seeing they are very low on resources.

**Reply by future2back** [reply]
Surely friend, this adjustment you mentioned would not be difficult for the dev team.
It already appears like this when we have low resource for cooking, but in red.
![image](https://user-images.githubusercontent.com/74167741/207971817-d2868c24-2438-484f-89f3-9e82cb7bac48.png)

This facilitates decision making and reduces the number of clicks and wasted time. I like it.
